modules/sdk-core/src/bitgo/tss/ecdsa/dkls-types.ts

```diff
@@ -0,0 +1,113 @@
+import * as t from 'io-ts';
+import { NonEmptyString } from 'io-ts-types';
+import { createStringObject } from '@bitgo-private/common-interface';
+
+/**
+ * Note these types are temporarily till we move types to wallet-platform-types from WP.
```

Comment:
https://github.com/BitGo/wallet-platform-types/blob/master/src/schema/keyGeneration/MPCv2/messages.ts maybe these can be used and moved to a more generic location